package.json

```diff
@@ -13,6 +13,10 @@
     "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --public-path=assets --loader:.js=jsx"
   },
   "devDependencies": {
-    "esbuild-loader": "^3.0.1"
+    "esbuild-loader": "^3.0.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
   }
```

Comment:
https://github.com/shella12/ReactRailsHello/blob/6c423127a2018982c36a4c42f1015e5550c361cf/package.json#L13
- [ ] NOTE: I can see that you used the esbuild loader dependency, however, `esbuild-loader` and `babel-loader` are not the same thing, although they are both Webpack loaders used for processing JavaScript code.
- [ ] Let's adhere to the project requirements 👍🏽 
- [ ] [REQUIRED]  Great job done @shella12 , however, It looks like you skipped some parts of the requirement. Take a look at a screenshot from the project requirement that I attached below:
<img width="778" alt="Screenshot 2023-04-21 at 10 46 09" src="https://user-images.githubusercontent.com/58692590/233605818-cc56044d-20bb-4109-b5e1-7ccc39cd3194.png">
Please ensure that you run this command :

``` ruby
yarn add @babel/core @babel/preset-env @babel/preset-react babel-loader --dev
```
and create a `babel.config.js` in the root folder of your project containing 

<img width="721" alt="Screenshot 2023-04-21 at 10 39 44" src="https://user-images.githubusercontent.com/58692590/233606490-520985b0-412f-40de-940b-42f8933f89ae.png">

- [ ] [REQUIRED] Let's follow the [tutorial](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a) as recommended so you can do the same for the `webpack.config.js` as well to include the babel-loader.

### Let's fix this 💪🏽